### ISSUE TITLE
"Committer identity unknown" error on v1.6.0

### ISSUE THREAD

**Issue by alyssa-glean** [EndBug/latest-tag#25]
The v1.6.0 release of this action has broken for us with an error about user config. We've pinned to v1.5.1 for the time being as a workaround. It's a private repo, but happy to share any other details that would be useful.

We're using self-hosted runners for our pipeline - not sure if that's relevant but the auto-detected email address (`'runner@ks-1cpu-1gi-v56-bxd79-bh2nj.(none)`) is the user and name of the pod running the job.

Our config is:

```
- name: Create tag
  uses: EndBug/latest-tag@v1.5.1
  with:
    ref: '${{ needs.prepare.outputs.VERSION }}'
    description: '${{ needs.prepare.outputs.VERSION }}'
```

v1.5.1 output (working):

```
Run EndBug/latest-tag@v1.5.1
  with:
    ref: 2.1.5601.2377
    description: 2.1.5601.2377
    force-branch: false
Setting up git user...
Using '2.1.5601.2377' as tag name.
Creating annotated tag...
Pushing updated tag to repo...
To https://github.com/glean-notes/glean
 * [new tag]         2.1.5601.2377 -> 2.1.5601.2377
```

v1.6.0 output (not working):

```
Run EndBug/latest-tag@latest
  with:
    ref: 2.1.5601.1720
    description: 2.1.5601.1720
    force-branch: false
    git-directory: /runner/_work/glean/glean

Running git commands within /runner/_work/glean/glean
Using '2.1.5601.1720' as tag name.
Setting up git user...
Creating annotated tag...
Error: Command failed: git -C /runner/_work/glean/glean tag -a -f -m "2.1.5601.1720" 2.1.5601.1720
Committer identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'runner@ks-1cpu-1gi-v56-bxd79-bh2nj.(none)')
```

**Comment by EndBug** [comment]
Hi, thanks for reporting this
I actually encountered this as well the first time i tested the action (on GitHub-hosted runners!), but wasn't able to reproduce it :/

I'm going to release a patch that adds some debug logs, since as of now there's much debug info to work with. I'm gonna get back to you soon with this ✌🏻

**Comment by EndBug** [comment]
Alright, it should be fixed now 🚀 (`v1.6.1`, `v1`, `latest`)
Thanks for the report 💖 

@all-contributors please add @alyssa-glean for their bug report
